examples/docker/docker-compose-full.yml

```diff
@@ -53,7 +79,7 @@ services:
       WASMCLOUD_ALLOW_FILE_LOAD: "true"
       WASMCLOUD_OCI_ALLOWED_INSECURE: registry:5000
       OTEL_TRACES_EXPORTER: otlp
-      OTEL_EXPORTER_OTLP_ENDPOINT: http://tempo:4318/v1/traces
+      OTEL_EXPORTER_OTLP_ENDPOINT: http://localhost:4318
```

Comment:
This is the default now, right? Do we need this anymore?

Reply:
Yep, I kept it there for no good reasons, I'll remove 👍